suite-native/module-accounts/src/components/AccountRenameForm.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { Box, Text, Button, VStack } from '@suite-native/atoms';
+import { TextInputField, Form, useForm } from '@suite-native/forms';
+import { yup } from '@trezor/validation';
+import {
+    accountsActions,
+    AccountsRootState,
+    selectAccountByKey,
+    selectAccountLabel,
+} from '@suite-common/wallet-core';
+
+type AccountRenameFormProps = {
+    accountKey: string;
+    onSubmit: () => void;
+};
+
+const MAX_LABEL_LENGTH = 30;
+
+const accountRenameValidationSchema = yup.object({
+    accountLabel: yup.string().required().max(MAX_LABEL_LENGTH),
+});
+type AccountImportFormValues = yup.InferType<typeof accountRenameValidationSchema>;
+
+export const AccountRenameForm = ({ accountKey, onSubmit }: AccountRenameFormProps) => {
+    const dispatch = useDispatch();
+    const account = useSelector((state: AccountsRootState) =>
```

Comment:
Avoid selecting whole account, but this shouldn't be neccessary after changes I suggested in `accountActions.ts`

Reply:
fixed in e44e7fee9